components/kyma-environment-broker/internal/runtimeversion/runtimeversion.go

```diff
@@ -1,21 +1,36 @@
 package runtimeversion
 
-import "github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+import (
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/broker"
+)
 
 type RuntimeVersionConfigurator struct {
-	defaultVersion string
-	accountMapping *AccountVersionMapping
+	defaultVersion        string
+	defaultPreviewVersion string
+	accountMapping        *AccountVersionMapping
 }
 
-func NewRuntimeVersionConfigurator(defaultVersion string, accountMapping *AccountVersionMapping) *RuntimeVersionConfigurator {
+func NewRuntimeVersionConfigurator(defaultVersion string, previewVersion string, accountMapping *AccountVersionMapping) *RuntimeVersionConfigurator {
 	return &RuntimeVersionConfigurator{
-		defaultVersion: defaultVersion,
-		accountMapping: accountMapping,
+		defaultVersion:        defaultVersion,
+		defaultPreviewVersion: previewVersion,
+		accountMapping:        accountMapping,
 	}
 }
 
 func (rvc *RuntimeVersionConfigurator) ForProvisioning(op internal.ProvisioningOperation) (*internal.RuntimeVersionData, error) {
+
 	pp := op.ProvisioningParameters
+
+	// TODO: Clarify: what about FromAccountMapping?
```

Comment:
@piotrmiskiewicz Do we need to take care of FromAccountMapping?

Reply:
let's implement it

Reply:
please remember about the order. The higher priority has a value provided in provisioning parameters, then the mapping, at the end - default value (from an envionment value)